src/ssh/azext_ssh/__init__.py

```diff
@@ -0,0 +1,70 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+from knack.help_files import helps
+
+helps['ssh'] = """
+    type: group
+    short-summary: SSH into Azure VMs
+"""
+
+helps['ssh vm'] = """
+    type: command
+    short-summary: SSH into Azure VMs
+    examples:
+        - name: Give a resource group and VM to SSH to
+          text: |
+            az ssh vm --resource-group [group] --name [vmName] : az ssh vm --resource-group mygroup --name myvm
```

Comment:
Normally we do not need the part before `:`. Same with other examples. I think it's better to remove it to align with other modules/extensions.